lib/web_ui/test/canvaskit/fallback_fonts_golden_test.dart

```diff
@@ -123,6 +123,102 @@ void testMain() {
       // TODO: https://github.com/flutter/flutter/issues/71520
     }, skip: isIosSafari || isFirefox);
 
+    test('will put the Noto Emoji font before other fallback fonts in the list',
+        () async {
```

Comment:
Should this test call `FontFallbackData.debugReset` before everthing else?

Reply:
We already have `FontFallbackData.debugReset` in the `setUp`